static/app/views/projectDetail/charts/projectSessionsChartRequest.tsx

```diff
@@ -124,19 +124,23 @@ class ProjectSessionsChartRequest extends Component<
           query: queryParams,
         }),
       ];
-      // for users, we need to make a separate request to get the total count in period
-      if (displayMode === DisplayModes.STABILITY_USERS) {
+      // for crash free sessions and users, we need to make a separate request to get the total count in period
```

Comment:
@matejminar Is there a way to solve this in one request?

Reply:
you can ask API for more fields than one (so you can ask for count AND crash-free), but the problem is that the chart is requesting double the needed period (for the previous backdrop)

in theory, you could add count field to the request that top cards are making and use that